src/frontend/src/lib/components/transactions/AllTransactionsList.svelte

```diff
@@ -58,10 +65,8 @@
 
 <!--TODO: include skeleton for loading transactions and remove nullish checks-->
 {#if nonNullish(sortedTransactions) && sortedTransactions.length > 0}
```

Comment:
Maybe let's update `groupedTransaction` to be `undefined` if no `sortedTransactions` available, and then update this line to check `groupedTransaction` instead?

Reply:
good catch! done!